### ISSUE TITLE
Accessibility vs Security again 

### ISSUE THREAD

**Issue by iadawn** [w3c/wcag#2702]
I am concerned that 3.3.7 Accessible Authentication runs the same risk as, for example, 2.2.1 Timing Adjustable, in that any requirement in this area will simply be squashed by a 'security says no' response. For example, financial institutions regularly don't use whole passwords but just request a few characters from a longer password. This can't be completed by password managers at the moment but I don't know that security folks would be willing to step back from that pattern.

The field identification methods are already covered in 1.3.5 Input Purpose and I am not sure what else this SC would realistically achieve.

Broadly the focus and intent of it seems unclear.

**Comment by alastc** [comment]
Response for group review:

<hr>

> any requirement in this area will simply be squashed by a 'security says no' response.

That is possible, and not really within our scope.

We've had a security horizontal review, and it was not thought to be a problem from a security point of view.

> This can't be completed by password managers at the moment but I don't know that security folks would be willing to step back from that pattern.

These kind of things do take time to change, and the roll-out of WCAG 2.2 is likely to take some time for larger institutions. 

However, we can put a stake in the ground with the SC and say "this is how to do it accessibly", which isn't with remembering snippets of passwords.

Please note that one of the people involved in writing the understanding document were working with a bank at the time.

> I am not sure what else this SC would realistically achieve.

It isn't trying to ban username/password, but it does prevent things like one-time-codes that you have to type in. That was the initial use-case brought to the group by folks at the COGA TF.

In a complex area, what it is trying to achieve is outlined in the understanding doc, in particular: "There should be a path through authentication that does not rely on cognitive function tests. "

**Comment by camusamta** [comment]
Hey, @alastc - is there something on that use-case about one-time-codes from COGA TF you mention that I could take a look at?

**Comment by alastc** [comment]
They want a specific doc for this, but if you look at the COGA usable doc:
https://www.w3.org/TR/coga-usable/

And search for transcription, you’ll probably find something there. (I’d be more specific, but I’m travelling on a train at the moment, using phone.)

**Comment by camusamta** [comment]
Thank-you for that!

**Comment by alastc** [comment]
The [response above](https://github.com/w3c/wcag/issues/2702#issuecomment-1318962166) was [approved yesterday](https://www.w3.org/2023/01/31-ag-minutes#item10).
